### PR TITLE
Changes to Docs for current release

### PR DESCRIPTION
**Pull request by pecameron**
Existing docs are the originals from iptraf with some minor changes.
This is the first of a series of update to the documentation. There is a
lot left to do.

Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@vsamel Please take a look. I found the changes (had them locally and didn't push them). This was done quite a while ago and will likely need more changes. As I mentioned in the commit message there is a lot left to do.

File: CHANGES [hunk 1]
```diff
@@ -1,5 +1,12 @@
 CHANGES file for iptraf-ng
 
+* Philip Cameron <pcameron@redhat.com> - 1.1.5
```
Comment (vsamel) [inline CHANGES:3]: Postpone this to release commit (or commit/s just before release).

File: INSTALL [hunk 2]
```diff
@@ -1,181 +1,20 @@
...
+iptraf-ng uses the usual gcc based build environment. It also uses ncurses
+so that must be installed as well.
 
-   The file `configure.in' is used to create `configure' by a program
-called `autoconf'.  You only need `configure.in' if you want to change
-it or regenerate `configure' using a newer version of `autoconf'.
+This was built on an x86_64 running Fedora-23 which includes gcc 5.3.1
 
-The simplest way to compile this package is:
+$ git clone https://git.fedorahosted.org/git/iptraf-ng.git
```
Comment (vsamel) [inline INSTALL]: https://git.fedorahosted.org/git/iptraf-ng.git doesn't work anymore

File: Makefile [hunk 3]
```diff
@@ -163,8 +163,9 @@ rvnamed-o += src/getpath.o
 rvnamed-o += src/sockaddr.o
 rvnamed-o += src/usage.o
 
--include config.mak.autogen
--include config.mak
+# autoconf is not currently supported. It may come back later.
```
Comment (vsamel) [inline Makefile]: just delete all the autoconf/configure stuff

File: README [hunk 4]
```diff
@@ -1,14 +1,15 @@
 ==========================================================================
-IPTraf 3.0 README
+IPTraf-ng 1.1.5 README
```
Comment (vsamel) [inline README]: leave the version out of this

File: README [hunk 5]
```diff
@@ -1,14 +1,15 @@
...
 --------------------------------------------------------------------------
 See the RELEASE-NOTES for important update information.
+See the CHANGES file for the histroy of changes.
```
Comment (vsamel) [inline README]: histroy --> history

File: README.contact [hunk 6]
```diff
@@ -6,13 +6,15 @@ CONTACT INFORMATION
...
-(riker@seul.org). Fork was made by Nikola Pajkovsky(npajkovs@redhat.com).
+(riker@seul.org). Fork to IPTraf-ng was made by Nikola Pajkovsky(npajkovs@redhat.com).
+Current maintainer is Philip Cameron(pcameron@redhat.com)
 
 ===============
 Web information
 ~~~~~~~~~~~~~~~
-The official IPTraf sites are http://cebu.mozcom.com/riker/iptraf and
-http://iptraf.seul.org. IPTraf-ng sites is https://fedorahosted.org/iptraf-ng/
+IPTraf-ng sites are https://fedorahosted.org/iptraf-ng/
```
Comment (vsamel) [inline README.contact]: fedorahosted.org doesn't work anymore

File: src/iptraf.c [hunk 7]
```diff
@@ -252,6 +252,7 @@ static void program_interface(void)
 static const char *const iptraf_ng_usage[] = {
 	IPTRAF_NAME " [options]",
 	IPTRAF_NAME " [options] -B [-i <iface> | -d <iface> | -s <iface> | -z <iface> | -l <iface> | -g]",
+// PHIL need to add: -t timeout min | -L filename | -f | -h
```
Comment (vsamel) [inline src/iptraf.c]: changes not related to docs, leave it to another changeset

File: RELEASE-NOTES [hunk 8]
```diff
@@ -1,108 +1,25 @@
...
-	NEW FILTER BEHAVIOR
+	UPGRADING TO IPTRAF-NG
 	BUG FIXES
-	ADDITIONAL NETWORK INTERFACE SUPPORT
-	DOCUMENTATION FORMAT CHANGES
-	FILE FORMATS
-	CODE CHANGES
 
-UPGRADING TO IPTRAF 3.0
+UPGRADING TO IPTRAFNG 1.1.5
```
Comment (vsamel) [inline RELEASE-NOTES]: IPTRAFNG --> IPTRAF-NG

File: CHANGES (hunk 1 above)
Comment (pecameron) [reply CHANGES:3]: OK

File: INSTALL (hunk 2 above)
Comment (pecameron) [reply INSTALL]: fixed

File: Makefile (hunk 3 above)
Comment (pecameron) [reply Makefile]: OK

File: README (hunk 4 above)
Comment (pecameron) [reply README]: OK

File: README (hunk 5 above)
Comment (pecameron) [reply README]: fixed

File: README.contact (hunk 6 above)
Comment (pecameron) [reply README.contact]: Deleted

File: RELEASE-NOTES (hunk 8 above)
Comment (pecameron) [reply RELEASE-NOTES]: Fixed

File: src/iptraf.c (hunk 7 above)
Comment (pecameron) [reply src/iptraf.c]: deleted line

**Comment by pecameron** [comment]
@vsamel I made your suggested changes and will hold off the merge until after PR7 goes in,
